### PR TITLE
[1LP][RFR] test snapshot operations on powered off vm

### PR DESCRIPTION
**Pull request by apagac**
This is a new test for testing basic snapshot operations on a powered off vm.

Snapshot create, revert and delete.

### CODE REVIEW

**Comment by apagac** [comment]
Still references selenium. This will change after PR #5124 merge.

**Comment by apagac** [comment]
PRT is failing on unrelated test, this will go away with PR #5126 merge.

**Comment by apagac** [comment]
Now waiting for PR #5124 that contains fix for NameError and snapshot.refresh.

**Comment by apagac** [comment]
PRT 5.7 fails on unrelated test, so moving this to RFR.

Dependent PR was merged, this was rebased and refactored.

File: cfme/tests/infrastructure/test_snapshot.py [hunk 1]
```diff
@@ -220,6 +220,30 @@ def test_operations_suspended_vm(small_test_vm, provider, soft_assert):
 
 
 @pytest.mark.uncollectif(lambda provider: not provider.one_of(VMwareProvider))
+def test_operations_powered_off_vm(small_test_vm, provider, soft_assert):
+    # Make sure the VM is off
+    small_test_vm.power_control_from_cfme(option=small_test_vm.POWER_OFF, cancel=False)
+    small_test_vm.wait_for_vm_state_change(desired_state=small_test_vm.STATE_OFF)
+    # Create first snapshot
+    snapshot1 = new_snapshot(small_test_vm)
```
Comment (RonnyPfannschmidt) [inline cfme/tests/infrastructure/test_snapshot.py]: question for second level review: should we consider having a helper for the creation of a snapshot as a followup with a more nice api as this test needs to do quite some plumbing due to low level of the api provided

Comment (mfalesni) [reply cfme/tests/infrastructure/test_snapshot.py]: Shouldn't this be resolved with collections? You should then be able to do something like `vm.snapshots.create(...)` ...

Comment (RonnyPfannschmidt) [reply cfme/tests/infrastructure/test_snapshot.py]: @mfalesni perfect point, thanks 👍

**Pull request merged**